src/main/kotlin/frc/robot/subsystems/ExampleSubsystem.kt

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems
+
+import edu.wpi.first.wpilibj.DigitalInput
+import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.SubsystemBase
+
+/** Creates a new ExampleSubsystem.  */
+class ExampleSubsystem(
+    exampleConstructorArg: Double,
+    /**
+     * An example of a property that is also a constructor argument.
+     * The value of inputID depends on the value passed in
+     * when the ExampleSubsystem is created.
+     */
+    private val digitalInputID: Int,
+    private val motorID: Int,
```

Comment:
motorID is a good example of a constructor arg that doesn't need to be a property. It is used to construct the motor, but nothing else